src/main/java/com/icthh/xm/gate/GateApp.java

```diff
@@ -22,7 +23,8 @@
 import java.util.Arrays;
 import java.util.Collection;
 
-@SpringBootApplication(scanBasePackages = "com.icthh.xm")
+@SpringBootApplication(scanBasePackages = "com.icthh.xm",
```

Comment:
if Cassandra not needed, maybe just remove dependensies?

Reply:
Cassandra in gate app may be used as a storage for rate limiting feature. In our implementation we use hazelcast instead of cassandra. So I think cassandra can be removed

Reply:
Done